group_vars/google/packages.suse.yml

```diff
@@ -24,8 +24,8 @@
 ---
 packages:
   # rationale: Enables GCE platform features.
-  - google-guest-agent=20230601.00-150000.1.37.1
+  - google-guest-agent=20231031.01-150000.1.40.1
   # rationale: Support for GCE guest environment and functionality.
-  - google-guest-configs=20220211.00-150400.13.3.1
+  - google-guest-configs=20230808.00-150400.13.6.1
   # rationale: Google account authentication.
-  - google-guest-oslogin=20220721.00-150000.1.30.1
+  - google-guest-oslogin=20231101.00-150000.1.35.1
```

Comment:
Was this changing vendor, or changing to the openSUSE repos by chance? The version jump seems pretty large.

Reply:
Those are from SLES, not openSUSE.